lib/ruote/mon/storage.rb

```diff
@@ -273,8 +271,7 @@ def paginate(cursor, opts)
 
       return cursor.count if opts['count']
 
-      cursor.sort(
-        '_id', opts['descending'] ? Mongo::DESCENDING : Mongo::ASCENDING)
+      cursor.sort(:"_id" => opts['descending'] ? Mongo::Index::DESCENDING : Mongo::Index::ASCENDING)
```

Comment:
These constants are a little awkward here. Let's change this to `opts['descending'] ? -1 : 1`.